GameServerUnitTests/CommandsTests/WaltToCommandTests.cs

```diff
@@ -0,0 +1,20 @@
+﻿using GameServer.Commands;
+
+namespace GameServerUnitTests.CommandsTests
+{
+    [TestClass]
+    public class WaltToCommandTests
```

Comment:
typo =>Wal"t"

Reply:
Fixed the typo for both class name and .cs file